apps/vaporgui/QRangeSlider.cpp

```diff
@@ -32,6 +32,25 @@ QRangeSlider::QRangeSlider(Qt::Orientation orientation) : QSlider(orientation)
     this->setRange(0, QT_STOPS);
     this->setTracking(true);
     this->QSlider::setStyle(new QForceAbsoluteSetButtonsEnabledStyle(style()));
+
+// Fix for Qt bug https://bugreports.qt.io/browse/QTBUG-98093
+// Apply a style sheet to QSlider to make it work on OSX Monterey
+// Note: Inheriting from QMontereySlider does not work, so we manually apply the styleSheet
+#ifdef Darwin
+    this->setStyleSheet("\
```

Comment:
Does this replace the functionality of the above setStyle?

Reply:
If you are referring to the style changes when a handle is clicked, then yes.  The handle that is selected does not change its style.  But it functions just the same.

Would it help if I did a screen recording of what it looks like?

Reply:
The above setStyle changes the functionality of the sliders. It allows you to drag the bar rather than the controls.

Reply:
That's always been a feature.  Do you mean something else?

Reply:
I don't quite understand, sorry

Reply:
No problem - In the GIF below I'm dragging the bar with the sliders on this branch, and then I'm dragging the bar on the sliders on main.  In both cases, we can drag the bar so the functionality is unchanged.  It just looks different.

![rangeSlider](https://user-images.githubusercontent.com/9522770/151825250-d1d1fd4f-3103-4f1c-8991-83442957b3a8.gif)

Reply:
Ok great, that's what I was asking but I didn't quite understand your last reply. Thanks!